### PR TITLE
Revise 0316 - Clarification of transport switching rules and improvement of UX

### PR DESCRIPTION
**Pull request by Akihiro-Miyazaki**
## Introduction

This is an update of SDL-0316.
Update `Introduction` section according to [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/1069#issuecomment-689107562).

## Proposed solution
Change the word from "transport" to "primary transport".
Update the RPC `OnDeviceSwitchStatus` to add the parameters.
Modify Figure 1 according to [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/1069#issuecomment-689107562).

## Impact on existing code
Update `Impact on existing code` section according to [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/1069#issuecomment-689107562).

### CODE REVIEW

**Comment by Akihiro-Miyazaki** [comment]
Hi, @theresalech   -san.
This documentation was revised according to [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/1069#issuecomment-689107562).
Could you please review my proposal.
Thank you.

File: proposals/0316-Clarification-of-transport-switching-rules-and-improvement-of-UX.md [hunk 1]
```diff
@@ -6,60 +6,72 @@
...
+*1 The switching from BT to USB logic in the same device primary transport of iOS is already approved and implemented by the SDLC in SDL 0053 - Connectivity via iAP-BT and Transport Switch. See [PR](https://github.com/smartdevicelink/sdl_evolution/issues/158) and [sequence](https://cloud.githubusercontent.com/assets/11158516/26067529/3f428434-39a3-11e7-910b-6cc4069e0be3.png).<br>
 *2 With the current SDL standard specification, the device cannot identify whether it is the same device or not. Therefore, device identification by the HU must be a prerequisite.
 
 <b>Figure 1.</b> Sequence with new RPC added to existing logic
 
-![Figure_1_Sequence_with_new_RPC_added_to_existing_logic.png](../assets/proposals/0316-Clarification-of-transport-switching-rules-and-improvement-of-UX/Figure_1_Sequence_with_new_RPC_added_to_existing_logic.png)
+![Figure_1_Sequence_with_new_RPC_added_to_existing_logic.png](../assets/proposals/0316-Clarification-of-transport-switching-rules-and-improvement-of-UX/Figure_1_Sequence_with_new_RPC_added_to_existing_logic1.png)
```
Comment (theresalech) [inline proposals/0316-Clarification-of-transport-switching-rules-and-improvement-of-UX.md:74]: I've found a few required changes to this image:
- "unregisteration" should be "unregistration" in bottom right corner.
- "Guard" in bottom left corner should be removed.
- "Reconnection timer IS expired" condition should be added to bottom left corner (see: https://cloud.githubusercontent.com/assets/11158516/26067529/3f428434-39a3-11e7-910b-6cc4069e0be3.png).

**Review by theresalech — CHANGES_REQUESTED** [review]
Hi @Akihiro-Miyazaki, thanks for your PR! I have left one comment requesting changes to Figure 1. Please let me know if you have any questions or concerns with addressing this feedback. Thank you!

File: proposals/0316-Clarification-of-transport-switching-rules-and-improvement-of-UX.md (hunk 1 above)
Comment (Akihiro-Miyazaki) [reply proposals/0316-Clarification-of-transport-switching-rules-and-improvement-of-UX.md:74]: @theresalech -san,
Thank you for your advise.
I updated the image file.
Could you please confirm it.
Thank you.

**Review by theresalech — APPROVED** [review]

**Pull request merged**